aspnet/mvc/views/layout.rst

```diff
@@ -0,0 +1,98 @@
+Layout
+=======
+By `Steve Smith`_
+
+Views can take advantage of Layout, ViewStart, and ViewImports files to share common elements.
+
+.. contents:: Sections
+	:local:
+	:depth: 1
+	
+`View sample files <https://github.com/aspnet/Docs/tree/master/aspnet/mvc/views/layout/sample>`_
+
+What is a Layout
+----------------
+
+Most web apps have a common layout that provides the user with a consistent experience as they navigate from page to page. The layout typically includes common user interface elements such as the app header, navigation or menu elements, and footer. 
+
+.. image:: layout/_static/page-layout.png
+
+Common HTML structures such as scripts and stylesheets are also frequently used by many pages within an app. All of these shared elements may be defined in a *layout* file, which can then be referenced by any view used within the app. Layouts reduce duplicate code in views, helping them follow the `Don't Repeat Yourself (DRY) principle <http://deviq.com/don-t-repeat-yourself/>`_.
+
+The default Visual Studio ASP.NET MVC project template defines a layout file named ``_Layout.cshtml`` in the ``Views/Shared`` folder:
+
+.. image:: layout/_static/web-project-views.png
+
+This layout defines a top level template for views in the app. Apps do not require a layout, and apps can define more than one layout, with different views specifying different layouts.
+
+An example ``_Layout.cshtml``:
+
+.. literalinclude:: ../../../common/samples/WebApplication1/src/WebApplication1/Views/Shared/_Layout.cshtml
+  :language: html
+  :emphasize-lines: 42,66
+
+Specifying a Layout
+-------------------
+
+Razor views have a ``Layout`` property. Individual views specify a layout by defining this property:
```

Comment:
Ehh, you can technically override it. But nm, leave this as is. That'd be jumping down too much of a rabbit hole.